src/myaml/converters/utils/line.py

```diff
@@ -21,13 +22,13 @@ def get_lines(string):
     return lines
 
 
-def line_from_string(string):
+def line_from_string(string: str) -> Line:
     indent = get_indent(string=string)
     value = get_string(string=string)
     return Line(string=value, indent=indent)
 
 
-def get_indent(string):
+def get_indent(string: str) -> int:
```

Comment:
I need to fix this so that `- ` will have an indent of 1.
This is so yaml like
```
key:
- value
- value2
```
is supported

Reply:
is this enough? Is this the right place?